dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/viewtools/XmlToolTest.java

```diff
@@ -0,0 +1,53 @@
+package com.dotcms.rendering.velocity.viewtools;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.liferay.util.StringPool;
+import graphql.AssertException;
+import java.io.File;
+import java.net.URL;
+import java.util.Iterator;
+import org.junit.Test;
+
+public class XmlToolTest {
```

Comment:
this needs to be added to MainSuite so it runs.

Reply:
done https://github.com/dotCMS/core/pull/21460/commits/a499fb6a2476b0f8dce9d96b52a742cff0dc212e